packages/ketcher-react/src/templates/salts-and-solvents.sdf

```diff
@@ -3642,7 +3642,7 @@ methyl ethyl ketone
 >  <abbreviation>
 MEK
 
->  <group>
+>  <group>sa
```

Comment:
Is it supposed to be here?

Reply:
wow! good catch! it does not break things though for some reason